modules/rosapps/applications/screensavers/mazescr/maze.c

```diff
@@ -632,7 +632,7 @@ static BOOL OnCreate(HWND hWnd, LPCREATESTRUCT lpCreateStruct)
     return TRUE;
 }
 
-BOOL WINAPI AboutProc(HWND hWnd, UINT message, WPARAM wparam, LPARAM lparam)
+INT_PTR WINAPI AboutProc(HWND hWnd, UINT message, WPARAM wparam, LPARAM lparam)
```

Comment:
```suggestion
INT_PTR CALLBACK AboutProc(HWND hWnd, UINT message, WPARAM wparam, LPARAM lparam)
```

Reply:
What would be the benefit of that change?

Reply:
Saw your comment in the other ticket: Logically you said: "This is no Windows API, not exported". Ok then. Suggestion committed.